var/spack/repos/builtin/packages/py-kosh/package.py

```diff
@@ -28,3 +30,13 @@ class PyKosh(PythonPackage):
     depends_on("py-llnl-sina@1.11:", type=("build", "run"))
     depends_on("py-networkx", type=("build", "run"))
     depends_on("py-numpy", type=("build", "run"))
```

Comment:
Need to update networkx/numpy required versions

Reply:
The dependencies are unversioned in requirements.txt, so they likely need to be done there as well. @doutriaux1 do you know what versions kosh supports?  I know some of our simulators are stuck on old versions of numpy.  Have there been any problems with LLNL software stacks stuck on numpy 1.13 or numpy 1.8 ?

My understanding is that dependent packages like h5py and hdbscan have more stringent requirements.

Reply:
If think hdbscan (or maybe scipy?) requires a greater numpy (or maybe scipy?). I know Lina Muryanto had to movr numpy to 1.24, which forced h5py to >3. But it might have been to make idesign happy. I will check with her.

Reply:
in kosh@3.0, hdbscan@0.8.29 requires numpy@1.20:. Will that also require an h5py upgrade?

Reply:
Found the reqs in setup.py, fixed.